documentation/examples/rbac-setup.yml

```diff
@@ -22,6 +22,7 @@ rules:
   - extensions
   resources:
   - ingresses
+  - networking.k8s.io
```

Comment:
I believe `networking.k8s.io` is an apiGroup, not a resource.

Reply:
sorry, I have modifed this error